avocado/core/job.py

```diff
@@ -338,7 +338,7 @@ def _update_latest_link(self):
         """
         def soft_abort(msg):
             """ Only log the problem """
-            LOG_JOB.warning("Unable to update the latest link: {}", msg)
+            LOG_JOB.warning("Unable to update the latest link: %s", msg)
```

Comment:
I'm having a hard time trying to understand why the previous commit caused the linter to flag this (I've bisected it to a8f60853ef0975587f5de7c445d256f3d13286f8).

Anyway, if we understand it or not, I'd still either incorporate this into the previous commit, or change the order so that future bisections running `make check` would not falsely flag a8f60853ef0975587f5de7c445d256f3d13286f8.

Reply:
For some reason when calling `TestSuite.from_config(self.config)` without the `runner`, the error msg disappeared. :-O
Makes no sense to me.

I will revert this.